.prettierignore

```diff
@@ -1,6 +1,9 @@
 /coverage
 **/*/api-docs
+/docs/apidocs
+/docs/site/apidocs
 packages/cli/generators/*/templates
+packages/tsdocs/fixtures
```

Comment:
What is the reasoning for excluding the source code in `fixtures` directory from Prettier? AFAICT, those files contain regular TypeScript source, we should format those files the same way as the rest of the monorepo.